blocks/v2-video/v2-video.js

```diff
@@ -0,0 +1,72 @@
+import { createElement } from '../../scripts/scripts.js';
+
+const onHoverOrScroll = (element, handler) => {
+  let isInViewport = false;
+  let isMouseOver = false;
+  const onChange = () => {
+    handler(isInViewport || isMouseOver);
+  };
+
+  const observer = new IntersectionObserver((entries) => {
+    entries.forEach((entry) => {
+      if (entry.isIntersecting) {
+        isInViewport = entry.intersectionRatio >= 0.2;
```

Comment:
This intersection ratio is too small, I’d rather put it at 0.5

Reply:
Changed to 0.5